roles/overpass-api/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>api.openstreetmap.fr</title>
+  <style>
+    @import url(https://fonts.googleapis.com/css?family=Ubuntu+Condensed);
```

Comment:
I pick this font becausi it seems to be one used on http://openstreetmap.fr/ but it's not really important.